src/Mono.Android/Android.Runtime/JNINativeWrapper.cs

```diff
@@ -73,10 +73,10 @@ public static Delegate CreateDelegate (Delegate dlg)
 			ig.Emit (OpCodes.Leave, label);
 
 			bool  filter = Debugger.IsAttached || !JNIEnvInit.PropagateExceptions;
-			if (filter && AndroidRuntimeInternal.mono_unhandled_exception_method != null) {
+			if (filter && AndroidRuntimeInternal.mono_unhandled_exception.Method != null) {
```

Comment:
I don't think `Action<Exception>.Method` can be null? It's not a nullable `MethodInfo?`:

https://source.dot.net/#System.Private.CoreLib/src/libraries/System.Private.CoreLib/src/System/Delegate.cs,69

So maybe the only check here is for `filter`?

Reply:
Makes sense thanks, applied